examples/src/python/example/python_distribution/hello/pants_setup_requires/BUILD

```diff
@@ -0,0 +1,18 @@
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+main_file = 'main.py'
+
+python_dist(
+  sources=rglobs('*.py', exclude=[main_file]),
+  setup_requires=[
+    # FIXME: this currently fails because setup_requires doesn't fetch transitive deps!
```

Comment:
Still true?

As an aside - I'm super-uncomfortable with these FIXMEs hitting master. It may be just a vocabulary difference though. TODO (+ issue) - great, FIXME broken - like this implies - not great.

Reply:
It's a bad habit I picked up reading LLVM sources a while ago, I have zero issues with never seeing it again.

Reply:
This line was removed because it was untrue now that the rest of this PR works, I turned a couple nearby FIXMEs into TODOs, and moving forward will avoid the use of FIXME altogether.